pkg/users/service.go

```diff
@@ -0,0 +1,40 @@
+package users
+
+import (
+	"fmt"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
+)
+
+// Service specifies an API to the domain service.
+type Service interface {
+	// Authenticate validates the given token. If it is valid,
+	// nil error value is returned. If the token in invalid,
+	// a non-nil error value is returned.
+	Authenticate(token string) error
+}
+
+var _ Service = (*service)(nil)
```

Comment:
I found it hard to understand this: `var _ Service = (*service)(nil)`. Is it necessary?

It seems to check statically (compiler time) if the interface is really implemented. Even finding good explanations [here](https://stackoverflow.com/a/13194635) and [here](https://www.reddit.com/r/golang/comments/8zw964/why_would_you_declare_var_with_blank_identify/), there's no consensus about if it's a good or bad practice.

As a matter of personal preference, I wouldn't go with that.

Reply:
Done :smile: